Mutagen.Bethesda.Analyzers.Skyrim/Record/Key/WeightValueAnalyzer.cs

```diff
@@ -21,6 +21,7 @@ public class WeightValueAnalyzer : IIsolatedRecordAnalyzer<IKeyGetter>
     public void AnalyzeRecord(IsolatedRecordAnalyzerParams<IKeyGetter> param)
     {
         var key = param.Record;
+        if (key.IsDeleted) return;
```

Comment:
Hmm, do we perhaps want to consider not running isolated analyzers on deleted records?  Contextual would ofc need to handle themselves, still.

No isolated record analyzers should really be doing much on a deleted record.  We'd maybe make one specialized analyzer to check if any deleted record had context which would be a warning (it shouldnt have any)

Reply:
I thought about that too, but then wasn't sure if there might be some analyzers that still want to do it for some reason. But I think I'd welcome the change if we can have a variant that can also check deleted records if necessary. I always forgot this check and it was annoying :D